web/package.json

```diff
@@ -1,43 +0,0 @@
-{
-  "name": "@gravitational/namespaces",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "Apache-2.0",
-  "scripts": {
-    "test": "jest",
-    "test-coverage": "jest --coverage && scripts/print-coverage-link.sh",
-    "test-update-snapshot": "yarn test -- --updateSnapshot",
-    "tdd": "jest --watch",
-    "package-term": "yarn workspace @gravitational/teleterm package",
-    "package-term-ci": "yarn workspace @gravitational/teleterm package-ci",
-    "build-term": "yarn workspace @gravitational/teleterm build",
-    "build-native-deps-for-term": "yarn workspace @gravitational/teleterm build-native-deps",
-    "build-and-package-term": "yarn build-term && yarn package-term",
-    "build-and-package-term-ci": "yarn build-term && yarn package-term-ci",
-    "start-term": "yarn workspace @gravitational/teleterm start",
-    "start-teleport": "yarn workspace @gravitational/teleport start",
-    "start-teleport-e": "yarn workspace @gravitational/teleport.e start",
-    "build-teleport": "yarn build-teleport-oss && yarn build-teleport-e",
-    "build-teleport-oss": "yarn workspace @gravitational/teleport build --output-path=../../../webassets/teleport/app",
-    "build-teleport-e": "yarn workspace @gravitational/teleport.e build --output-path=../../../webassets/e/teleport/app",
-    "build-oss": "yarn build-teleport-oss",
-    "build-e": "yarn build-teleport-e",
-    "nop": "exit 0"
```

Comment:
@ryanclark `development` uses the `nop` target as the default `NPM_SCRIPT`. Just a note that we'll want to remove it from there...

Reply:
I'll leave it then in the root package.json. AFAIK there's no easy way around it when using a Dockerfile.